pkg/scheduler/framework/preemption/preemption.go

```diff
@@ -355,18 +359,26 @@ func (ev *Evaluator) prepareCandidate(ctx context.Context, c Candidate, pod *v1.
 				if apipod.UpdatePodCondition(newStatus, condition) {
 					if err := util.PatchPodStatus(ctx, cs, victim, newStatus); err != nil {
 						klog.ErrorS(err, "Preparing pod preemption", "pod", klog.KObj(victim), "preemptor", klog.KObj(pod))
-						return framework.AsStatus(err)
+						errCh.SendError(err)
```

Comment:
`SendErrorWithCancel` might be better.

Reply:
One example of `SendErrorWithCancel` you can refer is ↓
https://github.com/kubernetes/kubernetes/blob/fd358d360618bcc30ed9579a9cfd25c7fdc293f4/pkg/scheduler/framework/runtime/framework.go#L900